src/builtin-addons/core/template-completion-provider.ts

```diff
@@ -379,10 +381,13 @@ export default class TemplateCompletionProvider {
             });
 
           roots.forEach((r) => {
+            const rootNameParts = r.name.split(path.sep);
```

Comment:
name could not be windows-like (\\), so, we don't need path.sep here, it always will be "/"

Reply:
fixed